specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/chat_thread_created_withUser.json

```diff
@@ -41,10 +44,16 @@
           "rawId": "8:acs:3d703c91-9657-4b3f-b19c-ef9d53f99710_00000008-576c-289b-07fd-0848220015ea",
           "communicationUser": {
             "id": "8:acs:3d703c91-9657-4b3f-b19c-ef9d53f99710_00000008-576c-289b-07fd-0848220015ea"
+          },
+          "metadata": {
+            "botReplyTo": "any"
```

Comment:
This is specific to an internal feature, let's change the example to something more generic